presto-main/src/main/java/com/facebook/presto/server/TaskResource.java

```diff
@@ -350,6 +407,12 @@ public void abortResults(@PathParam("taskId") TaskId taskId, @PathParam("bufferI
         taskManager.abortTaskResults(taskId, bufferId);
     }
 
+    /**
+     * the last worker-to-coordinator shuffle related
+     *
+     * The coordinator will shutdown remote sources to prevent a burst of requests.
```

Comment:
I'm not sure I understand this. Would you clarify? I wonder if we should implement this API in the C++ worker. Currently it is a no-op.